src/thunderbots/software/ai/hl/stp/tactic/tactic.h

```diff
@@ -18,19 +19,35 @@ class Tactic
 {
```

Comment:
Could we give a high level overview in the javadoc function for this class of how we intend it to be used? (most concrete implementations will have an `updateParams` function, etc.)

Reply:
comment added